GsaGH/GsaGH.csproj

```diff
@@ -44,7 +44,7 @@ Copyright © Oasys 1985 - 2022</Description>
     <PackageReference Include="Grasshopper" Version="6.27.20176.5001" IncludeAssets="compile;build">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="OasysGH" Version="0.4.4-beta" />
+    <PackageReference Include="OasysGH" Version="0.4.5-beta" />
```

Comment:
is that related to the pr headline? also, we have to change the other two plugins at the same time!